src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -67,7 +67,7 @@ export class GithubAppsLogsComponent implements OnInit {
     this.columnsToDisplay = this.matDialogData.userId
       ? ['organization', 'repository', 'reference', 'success', 'type']
       : ['repository', 'reference', 'success', 'type'];
-    this.displayedColumns = ['eventDate', 'githubUsername', ...this.columnsToDisplay];
+    this.displayedColumns = ['eventDate', 'githubUsername', 'entryName', 'deliveryId', ...this.columnsToDisplay];
```

Comment:
I don't think `deliveryId` will be of much use to our users, and is just cluttering up the dialog for them; I'd argue we could hide it for non-admins/curators. I'll defer to others' opinions.

Reply:
I like the idea of displaying the id just for us. I checked and it doesn't look like regular endusers can debug payloads, just us.

Reply:
It might be useful if users want to see which entries were affected by the same webhook invocation. Example: broad repo updates and many workflows were updated, resulting in many lambda events. User might try to use the delivery ID to filter for specific lambda events